packages/typescript/internal/ts_project.bzl

```diff
@@ -597,7 +597,7 @@ def ts_project_macro(
         write_tsconfig(
             name = "_gen_tsconfig_%s" % name,
             config = tsconfig,
-            files = srcs,
+            files = [s for s in srcs if _is_ts_src(s, allow_js)],
```

Comment:
Is now a good time to add ts_src_extensions and js_src_extensions to attributes and make it configurable?

Reply:
hmm what other values do you think a user might want to provide? I can't see why they would want such a config